.github/workflows/release.yml

```diff
@@ -0,0 +1,74 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+env:
+  GOPROXY: https://proxy.golang.org/
+  PROTOCOL_VERSION: "6.0"
+  PROTOCOL_VERSIONS: "6.0"
+  SIGNER: interim_signing_subkey_7685B676
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Read go version
+        id: go-version
+        run: |
+          content=`cat ./.go-version`
+          echo "::set-output name=content::$content"
+      - uses: actions/setup-go@v2
+        with:
+          # TODO: Replace with go-version-from-file when it is supported
+          # https://github.com/actions/setup-go/pull/62
+          go-version: ${{ steps.go-version.outputs.content }}
+      - uses: hashicorp/setup-hc-releases@v1
+        with:
+          github-token: ${{ secrets.HC_RELEASES_TOKEN }}
+          signer: ${{ env.SIGNER }}
+      - uses: hashicorp/setup-signore@v2
+        with:
+          github-token: ${{ secrets.SETUP_SIGNORE_GITHUB_TOKEN }}
+          signer: interim_signing_subkey_768B676
+      - name: Release Notes
+        run: sed -n -e "1{/# /d;}" -e "2{/^$/d;}" -e "/# \[$(git describe --abbrev=0 --exclude="$(git describe --abbrev=0 --match='v*.*.*' --tags)" --match='v*.*.*' --tags | tr -d v)/q;p" CHANGELOG.md > /tmp/RELEASE-NOTES.md
+      - name: Release Protocol Versions
+        run: |
+          manifestFile=${{ github.event.repository.name }}_manifest.json
+          if [[ -f  $manifestFile ]] && [[ $(jq '.metadata.protocol_versions' $manifestFile | jq length) -gt 0 ]]; then
+            versions=`jq '.metadata.protocol_versions' $manifestFile | jq 'sort'`
+            protoV=`echo $versions | jq '.[0]'`
+            protoVs=`echo $versions | jq 'join(", ")'`
+
+            echo "PROTOCOL_VERSION=${protoV}" >> $GITHUB_ENV
+            echo "PROTOCOL_VERSIONS=${protoVs}" >> $GITHUB_ENV
+          fi
```

Comment:
another thing i overlooked 😅 since this provider is built on SDK v2 (not plugin-framework..for now), then this step can be removed as it's specific to providers that use protocol 6 per https://www.terraform.io/docs/plugin/framework/publishing.html